### ISSUE TITLE
Weekly bug

### ISSUE THREAD

**Issue by zhex900** [jakubroztocil/rrule#222]
Hi,

This rule produces a bug.
````
let rule = {
  freq: RRule.WEEKLY,
  dtstart: start.toDate(),
  interval: 1,
  byweekday: [RRule.SU, RRule.WE],
  until: moment("10/10/2018", "DD/MM/YYYY").toDate()
};
````

The output have a date `Sat Oct 06 2018 00:00:00 GMT+1000`

Saturday should not be in the output. Is this a bug?

**Comment by davidgoli** [comment]
@zhex900 what is the value of `start`?

**Comment by davidgoli** [comment]
I believed this is fixed as of 2.3.4. Please reopen if you can repro.
